cloudevents/sdk/converters/binary.py

```diff
@@ -25,6 +25,13 @@ class BinaryHTTPCloudEventConverter(base.Converter):
     TYPE = "binary"
     SUPPORTED_VERSIONS = [v02.Event, ]
```

Comment:
Binary format was specified in v0.1 as well:

https://github.com/cloudevents/spec/blob/v0.1/http-transport-binding.md#31-binary-content-mode